ttnn/cpp/ttnn/operations/pool/maxpool/device/max_pool2d_multi_core_program_factory.cpp

```diff
@@ -289,7 +286,7 @@ MaxPool2D::MultiCore::cached_program_t max_pool_2d_multi_core_sharded_with_halo_
         kernel_size_w,
         pad_w,
         in_nbytes_c,
-        in_nbytes_c_log2,
+        in_nbytes_c_log2,   // not used
```

Comment:
Any reason why we can't remove this argument completely?

Reply:
also, can we remove this line?
```
    uint32_t in_nbytes_c_log2 = (uint32_t)std::log2((float)in_nbytes_c);
```

Reply:
This argument is still used by some kernels (there are multiple kernels which use this compute config), I will remove it as soon as the dependency is completely gone!

Reply:
if in this specific case it is not used - can you pass 0 or some other default value?

Reply:
I will remove this from the other kernels that use it so it can be removed completely.